conftest.py

```diff
@@ -0,0 +1,5 @@
+import sys
+import os
+
+src = os.path.abspath("filter")
+sys.path.insert(0, src)
```

Comment:
This is odd and we don't usually have to do this on any of our other repos. Why are you doing this?

Reply:
It's just the pattern we normally do, but I've changed it so this file is no longer required.